### PR TITLE
Sayalis/feature 7 month filter

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
@onkartibe  @aloka-cuelogic 

card link : https://trello.com/c/LLAfkNmG

Complete functionality.

### CODE REVIEW

**Comment by sayalis-cuelogic** [comment]
@onkartibe @aloka-cuelogic 
Resolved conflicts.
Please review the code.

File: post/views.py [hunk 1]
```diff
@@ -21,10 +21,32 @@ def get(self, request):
...
+                     "September": 9,
+                     "October": 10,
+                     "November": 11,
+                     "December": 12}
+
         year = request.POST.get('yearfilter')
-        post_data = {"postdata":
+        month = request.POST.get('monthfilter')
+        if year:
+            post_data = {"postdata":
```
Comment (aloka-cuelogic) [inline post/views.py]: Change this
Initialize a blank dict post_data = {}
based on conditions input value
like post_data['post_data'] = value

**Comment by sayalis-cuelogic** [comment]
Worked on review comments.
Used dict for sending post data.

**Comment by aloka-cuelogic** [comment]
Reviewed and Branch

**Pull request merged**